lib/data/prisoner_packs.yml

```diff
@@ -246,10 +246,10 @@
   lawyer: /government/publications/ireland-list-of-lawyers
 - slug: israel
   pdf: /government/publications/israel-prisoner-pack
-  lawyer: /government/world/israel/help-for-british-nationals/when-things-go-wrong/if-you-need-lawyer/
+  lawyer: /government/publications/israel-list-of-lawyers
```

Comment:
I'm not completely sure because I don't know too much about Smart Answers, but is it right that changing these `israel` and `italy` URLs *doesn't* change any test artefacts for them?

Reply:
The regression test coverage for `help-if-you-are-arrested-abroad` looks pretty limited:

```
gds8092 [msw-venezuela] >>>  ls -lR test/artefacts/help-if-you-are-arrested-abroad/
total 96
-rw-r--r--  1 michaelswalker  staff  3673 12 Jun 15:09 albania.txt
-rw-r--r--  1 michaelswalker  staff  3287 12 Jun 15:04 andorra.txt
-rw-r--r--  1 michaelswalker  staff  3923 12 Jun 15:04 austria.txt
-rw-r--r--  1 michaelswalker  staff  4023 12 Jun 15:04 cyprus.txt
-rw-r--r--  1 michaelswalker  staff  3765 12 Jun 15:04 democratic-republic-of-the-congo.txt
-rw-r--r--  1 michaelswalker  staff  3802 12 Jun 15:04 greece.txt
-rw-r--r--  1 michaelswalker  staff   113 12 Jun 15:04 help-if-you-are-arrested-abroad.txt
-rw-r--r--  1 michaelswalker  staff  3393 12 Jun 15:04 iran.txt
-rw-r--r--  1 michaelswalker  staff  3936 12 Jun 15:04 spain.txt
-rw-r--r--  1 michaelswalker  staff   798 12 Jun 15:04 syria.txt
-rw-r--r--  1 michaelswalker  staff  5836 12 Jun 15:04 y.txt
```

Reply:
Do you know how easy it would be to add them for Israel and Italy?

Reply:
Pretty straightforward, I think: https://github.com/alphagov/smart-answers/blob/master/test/data/help-if-you-are-arrested-abroad-questions-and-responses.yml
It looks like the current entries are there because they exercise different features.  Is it still worth adding `israel` and `italy`?

Reply:
Hmm, yeah, that sounds fine to leave out Israel and Italy.